src/core/js/views/drawerView.js

```diff
@@ -86,7 +86,14 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
             this.$('.drawer-holder').html(view);
+
+            var model = this.collection.findWhere({
```

Comment:
yes PLP has never had a title , but if we add title to the PLP object in course.json , what expected is it will show PLP title, 
expectation from this PR is  , if any(PLP,language picker,resource,glossary) drawer item has title then it shows title when we click on that particular drawer item,